src/modules/fw_att_control/FixedwingAttitudeControl.cpp

```diff
@@ -802,10 +803,18 @@ void FixedwingAttitudeControl::run()
 					}
 
 				} else {
-					// pure rate control
-					_roll_ctrl.set_bodyrate_setpoint(_rates_sp.roll);
+					vehicle_rates_setpoint_poll();
+
+					if (_parameters.vtol_type == vtol_type::TAILSITTER) {
```

Comment:
Can we handle all of these adjustments right after the orb_copy? Then the actual controller code can be generic.